test/library/standard/IO/readBinary/input/gen.txt

```diff
@@ -0,0 +1,117 @@
+// rust code used to generate binary files
```

Comment:
This file isn't one the test system is expected to run, right?  I might add a comment about what was generated by it (normally, I'd suggest including that in the generated file itself, but I think that would mess with your test programs)

Reply:
Yeah, I don't think I can include it in the files for this particular test, but I added an updated note to explain why it's there.